### PR TITLE
Create dir model

### PR DESCRIPTION
**Pull request by msrubar**
@jan-cerny found a bug in OVAL session described in PR #270. 

First commit adds fixes SIGSEGV which is caused when some OVAL Directives is used and second commit adds test catching this issue.

### CODE REVIEW

**Comment by jan-cerny** [comment]
Hi Míšo, you have forgotten to add new files to EXTRA_DIST variable in  `tests/API//OVAL/unittests/Makefile.am`.

File: tests/API/OVAL/unittests/test_directives.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+set -e -o pipefail
+
+name=$(basename $0 .sh)
+result=$(mktemp ${name}.out.XXXXXX)
+stderr=$(mktemp ${name}.err.XXXXXX)
+
+$OSCAP oval eval --results $result --directives $srcdir/$name.xml $srcdir/comment.xml 2> $stderr
+
+assert_exists 0 '/oval_results/oval_definitions'
```
Comment (jan-cerny) [inline tests/API/OVAL/unittests/test_directives.sh]: Is it possible have more asserts here? Does it make sense? Just asking :-)

**Comment by jan-cerny** [comment]
[test this please]

**Comment by jan-cerny** [comment]
[test this please]

**Comment by isimluk** [comment]
This test seems to fail in jenkins with
```
+ remove <oval_definiton> using oval directives               [ WARN ]
```
in https://jenkins.open-scap.org/job/openscap-pull-requests/300/console

**Comment by jan-cerny** [comment]
@isimluk  it is probably because of this:
> Hi Míšo, you have forgotten to add new files to EXTRA_DIST variable in tests/API//OVAL/unittests/Makefile.am.

**Comment by msrubar** [comment]
It should be ready now.

**Comment by jan-cerny** [comment]
@msrubar Thank you! LGTM. Happy new year!

**Pull request merged**